dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -195,3 +218,43 @@ properties:
         Selects the pulsed mode for data-ready interrupt when enabled,
         and the latched mode when disabled.
         Sets the corresponding DRDY_PULSED bit in the CTRL7 register.
+    
+    ff-duration:
+      type: int
+      default: 0x0
```

Comment:
Please document in the description why this particular default value was chosen

Reply:
The reason has been explained.